sys/androidmedia/gstamc.h

```diff
@@ -83,14 +83,18 @@ struct _GstAmcCrypto
 
 struct _GstAmcCodec
 {
+  guint flush_id;
+  GMutex buffers_lock;
   /* < private > */
   jobject object;               /* global reference */
+  volatile gint ref_count;
```

Comment:
This is not needed. No code is using it but something added by yourself. Please maintain implicit code guidelines.

Reply:
gint
g_atomic_int_add (volatile gint *atomic,
                  gint val);

Reply:
That's the method definition. Is a parameter qualifier. The member of the struct/class does not need to be volatile. Is just a compiler hint when that method is called.

Reply:
ok, done